SIL.DblBundle.Tests/Resources/TestBundleFiles/metadataVersion2_1.xml

```diff
@@ -13,7 +13,7 @@
       <fullName>Rabinal Achi NT (new alphabet) (Guatemala) (acrN) DBL</fullName>
     </systemId>
     <systemId type="reap">
-      <id>http://www.reap.insitehome.org/handle/9284745/10177</id>
+      <id>https://www.reap.sil.org/handle/9284745/10177</id>
```

Comment:
Are you sure https://www.reap.sil.org/ is correct? Doesn't resolve to an IP address on my machine. Based on the description in the PR I guess this should be https://reap.sil.org.